scipy/optimize/tests/test_direct.py

```diff
@@ -0,0 +1,67 @@
+"""
+Unit test for DIRECT optimization algorithm.
+"""
+from numpy.testing import (assert_, assert_array_almost_equal,
+                           assert_equal)
+from numpy.testing._private.utils import assert_almost_equal
```

Comment:
This will disappear following my other suggestions. The private was a hint that it should not have been used like this as well.